cmd/rootlesskit/main.go

```diff
@@ -108,24 +114,24 @@ See https://rootlesscontaine.rs/getting-started/common/ .
 		}, CategoryLXCUserNic),
 		Categorize(&cli.IntFlag{
 			Name:  "mtu",
-			Usage: "MTU for non-host network (default: 65520 for slirp4netns, 1500 for others)",
+			Usage: "MTU for non-host network (default: 65520 for pasta and slirp4netns, 1500 for others)",
 			Value: 0, // resolved into 65520 for slirp4netns, 1500 for others
 		}, CategoryNetwork),
 		Categorize(&cli.StringFlag{
 			Name:  "cidr",
-			Usage: "CIDR for slirp4netns network (default: 10.0.2.0/24)",
+			Usage: "CIDR for pasta and slirp4netns networks (default: 10.0.2.0/24)",
 		}, CategoryNetwork),
 		Categorize(&cli.StringFlag{
 			Name:  "ifname",
-			Usage: "Network interface name (default: tap0 for slirp4netns and vpnkit, eth0 for lxc-user-nic)",
+			Usage: "Network interface name (default: tap0 for pasta, slirp4netns, and vpnkit; eth0 for lxc-user-nic)",
```

Comment:
typo `Network interface name (default: tap0 for pasta, slirp4netns and vpnkit; eth0 for lxc-user-nic`

Reply:
Not [necessarily](https://en.wikipedia.org/wiki/Serial_comma#Recommendations_by_style_guides), but I couldn't find other examples in this file to check for consistency (other than [another one](https://github.com/rootless-containers/rootlesskit/pull/358/files/85b34d5037fac6f82bd8aee07c976f9f2e75e782#diff-a90a85572beea5c024965d62fc2abdce609e5dac6e7607fb344075515fc9a369R329) introduced by this commit itself).